<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget8BitLedLeiste</class>
 <widget class="QWidget" name="Widget8BitLedLeiste">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="led_7" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_6" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_5" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_4" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_3" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_2" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_1" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="led_0" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
